piksi_tools/console/baseline_view.py

```diff
@@ -165,18 +180,35 @@ def baseline_callback(self, sbp_msg):
       table.append(('GPS Time', t))
       table.append(('GPS Week', str(self.week)))
 
-      if self.log_file is None:
-        self.log_file = open(time.strftime("baseline_log_%Y%m%d-%H%M%S.csv"), 'w')
-        self.log_file.write('time,north(meters),east(meters),down(meters),distance(meters),num_sats,flags,num_hypothesis\n')
-
-      self.log_file.write('%s,%.4f,%.4f,%.4f,%.4f,%d,0x%02x,%d\n' % (
-        str(t),
-        soln.n, soln.e, soln.d, dist,
-        soln.n_sats,
-        soln.flags,
-        self.num_hyps)
-      )
-      self.log_file.flush()
+      if self.file_name_b == '':
+        if self.json:
```

Comment:
please remove the json logic here. As we discuessed file extension is always .csv for this file